pages/phases/job-announcement/create-job-announcement.md

```diff
@@ -26,26 +26,26 @@ Source: 5 U.S.C. § 5105 Standards for classification of positions
 
 ### 3. Duties
 
 Duties should be informed by the job tasks you documented during job analysis. Place the most important job duties as the first few bullets.
 
-If you are posting for multiple roles (this is not typical), and your SMEs determine in job analysis that applicants should only reasonably be allowed to apply for a subset of those roles, include this text in this section (modifying the specific number as appropriate):
+If you are posting for multiple speciality areas (not typical) and, during job analysis, your SMEs determine that applicants should be allowed to apply for one or more of those specialty areas, include the following language in this section (modifying the specific number as appropriate):
 
-> You may apply for up to two (2) roles using the Apply button at the bottom of the page. If you choose more than (2) you will only be considered for the first two selected in the order they are listed.
+> You may apply for up to two (2) specialty areas using the Apply button at the bottom of the page. If you choose more than (2) you will be considered only for the first two selected in the order they are listed.
 
 ### 4. Qualifications
 
-List all competencies you developed during job analysis together to describe the minimum qualifications for the position. If required for the position, you can describe specific recency requirements when skills obtained many years ago would not be relevant to the work being performed. For example, a technical position may require experience with modern cloud technologies within the last three years.
+List all competencies developed during job analysis to describe the minimum qualifications for the position. If required for the position, you can describe specific recency requirements when skills obtained many years ago would not be relevant to the work being performed. For example, a technical position may require experience with modern cloud technologies gained within the last three years.
 
-Include only the competencies SMEs developed during job analysis as the qualifications for your position. You should not be cutting and pasting the OPM Qualification Standards nor from the <a href="{{ site.baseurl }}/hiring-phases/getting-started/identifying-appropriate-positions/">Position Description</a>. As per OPM guidance, you are required to do the work of job analysis to define the specialized experience required, which in our pilots was in the form of competencies and proficiency levels.
+Include only the competencies SMEs developed during job analysis as the qualifications for your position. You should not cut and paste the OPM Qualification Standards nor from the Position Description. In accordance with Federal regulations and OPM guidance, you must conduct a job analysis to define the specialized experience required, which our pilots present in the form of competencies and proficiency levels.
```

Comment:
here, i think it is important to let readers (especially those from HR) know that SMEs are not doing this work alone.  instead, it is SMEs in tandem with HR developing competencies from job analysis.  HR needs to[explicitly] "see themselves" in the process.
```suggestion
Include only the competencies developed with SMEs during job analysis as the qualifications for your position. You should not cut and paste the OPM Qualification Standards nor from the Position Description. In accordance with Federal regulations and OPM guidance, you must conduct a job analysis to define the specialized experience required, which our pilots present in the form of competencies and proficiency levels.
```